backend/app/models.py

```diff
@@ -0,0 +1,20 @@
+from . import db
+
+
+class User(db.Model):
```

Comment:
Password should be added. It would be useful for providing endpoints for frontend.